streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/worker/context/StreamingRuntimeContext.java

```diff
@@ -0,0 +1,61 @@
+package org.ray.streaming.runtime.worker.context;
+
+import java.util.Map;
+import org.ray.streaming.api.context.RuntimeContext;
+import org.ray.streaming.runtime.core.graph.executiongraph.ExecutionVertex;
+import org.ray.streaming.util.Config;
+
+/**
+ * Use Ray to implement RuntimeContext.
+ */
+public class StreamingRuntimeContext implements RuntimeContext {
+  private int globalTaskId;
```

Comment:
Why name it as `globalTaskId`? Is there anything like `localTaskId`? If not, maybe taskId is enough

Reply:
Good idea, it has been modified.